.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "components/libesphttpd"]
 	path = components/libesphttpd
 	url = https://github.com/chmorgan/libesphttpd.git
+[submodule "components/mkspiffs"]
+	path = components/mkspiffs
+	url = https://github.com/phatpaul/mkspiffs.git
```

Comment:
any reason not to use the main GitHub repository for mkspiffs? Igrr's?

Reply:
I modified that project to get it to work as a submodule in ESP-IDF.  I don't know if there is a cleaner way. 
https://github.com/phatpaul/mkspiffs/commit/04d419f54f512494570ef7eef82c52013426948c
Thoughts?

Reply:
@phatpaul maybe reach out to igrr to see what he thinks? It may be that we can perform the configuration parts as a part of the build inside of this repository.

Reply:
The commit linked above seems to be the correct approach, if you want to include mkspiffs as a submodule.

Reply:
@igrr @phatpaul had to add this commit, https://github.com/phatpaul/mkspiffs/commit/04d419f54f512494570ef7eef82c52013426948c

Would that commit be accepted into your tree if submitted as a PR?

Reply:
Yes, it would.

Reply:
@phatpaul can you submit that commit as a PR? I tried but GitHub is reporting an error, probably because it requires some permissions on your fork.